spec/features/maker_log_out_spec.rb

```diff
@@ -0,0 +1,7 @@
+feature 'makers can log out to chitter' do
+  scenario 'successful log out when logged in' do
+    sign_up
+    click_button 'Log out'
+    expect(page).to have_content 'Not currently logged in'
+  end
```

Comment:
Could you create a log out button that is only visible when the user is logged in?